core/src/test/java/test/org/apache/spark/JavaTaskCompletionListenerImpl.java

```diff
@@ -15,9 +15,10 @@
  * limitations under the License.
  */
 
-package org.apache.spark.util;
+package test.org.apache.spark;
```

Comment:
note that I changed the package to make sure we test package visibility correctly also. 

We should do this for other Java API tests, but we can do those later.